### ISSUE TITLE
Parallel build?

### ISSUE THREAD

**Issue by sonic182** [tallakt/codepagex#29]
Would be nice to have the compilation in parallel, it takes too long for certain encodings and the fact that it uses only one core is worst

**Comment by sonic182** [comment]
I think the solution is to define various files (or modules) instead of a really BIG module, so it will compile in parallel the modules

**Comment by tallakt** [comment]
I am not finding time for this lately. I would gladly accept a pull request…
